scripts/zones/Chocobo_Circuit/npcs/odersille.lua

```diff
@@ -0,0 +1,22 @@
+-----------------------------------
+-- Area: Chocobo_Circuit
+-- NPC: Odersille
+-- Grandstand Exit
+-- pos -14.3511 -15 -131.869
+-----------------------------------
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
+end
+
+entity.onTrigger = function(player, npc)
+    player:startEvent(331)
+end
+
+entity.onEventUpdate = function(player, csid, option)
+end
+
+entity.onEventFinish = function(player, csid, option)
+end
+
+return entity
```

Comment:
As mention end file needs a new line

Sanity Check will let you know what needs to be fixed in most things

scripts/zones/Chocobo_Circuit/npcs/odersille.lua: No newline at end of file, please add one.

Reply:
I know. git editor or notepad++ are a pain. its hard to work with invisible characters.

Reply:
get the editorconfig plugin for notepad++ and a majority of its formatting problems get automated away.